### DISCUSSION TITLE
4 Questions Regarding Taxation

### DISCUSSION THREAD

**Discussion by y124** [Luke100000/minecraft-comes-alive#372] [Q&A]
## ~~4~~ 5 questions regarding taxation of villagers
1. does a villager have any prerequisites to be taxed? (eg. having a job, having a certain happiness ect.)
2. what is the average amount of emeralds per villager per week? (assume 100% tax)
3. how do i know how far through the week i am?
4. are there any ways to increase the amount of tax a villager pays other than increasing it in the rules? (eg. having a certain job, having a certain happiness ect.)
Edit:
5. do libraries stack and if so in what way (2 libraries 150%, 200% or 225% taxes) 
### Thank you so much in advance.

**Comment by Luke100000** [comment]
Sorry for this late response, I tend to miss discussions on GitHub :(

1) nope, not yet but especially the happiness should contribute to the chance of taxes being a success, thanks
2) configurable, and I dont want to mention a value here because its prone to changes
3) there will be a calendar soon for similar events, but currently, you dont really know
4) not yet, see 1)
5) no, for obvious reasons :) There will be additional buildings doing so, but they all will be limited to one per type.
